events/sinks/gcl/gcl.go

```diff
@@ -48,13 +49,18 @@ func (sink *gclSink) ExportEvents(eventBatch *core.EventBatch) {
 	glog.V(4).Info("Exporting events")
 	entries := make([]*gcl.LogEntry, len(eventBatch.Events))
 	for i, event := range eventBatch.Events {
+		evtJson, err := json.Marshal(event)
```

Comment:
The library updated from taking an `interface{}` type to taking a pre-encoded blob of json.  Presumably, it was just calling `json.Marshal` before, but the library was doing it internally.  I can switch it over to `runtime.Encode`